nxdrive/data/qml/Systray.qml

```diff
@@ -376,7 +376,7 @@ Item {
                 Layout.alignment: Qt.AlignHCenter
                 Layout.topMargin: 10
                 onClicked: {
-                    systray.hide()
+                    application.hide_systray()
```

Comment:
Is it related to https://jira.nuxeo.com/browse/NXDRIVE-1295? If yes, can you add a specific commit? As the ticket will need several commits, I prefer to have clear separated patches.